Kerberos.NET/Kerberos.NET.Simple.csproj

```diff
@@ -0,0 +1,94 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What's the purpose of this project file if the main one has both target frameworks set?

Reply:
Accidental checking. I originally thought about creating two project files, then realized i can just use a condition inside one project file and forgot to get rid of this. I'll delete